server/adaptors/integrations/__data__/repository/aws_waf/assets/create_mv-1.0.0.sql

```diff
@@ -0,0 +1,25 @@
+CREATE MATERIALIZED VIEW {table_name}_mview AS
+SELECT
+    `timestamp` AS `@timestamp`,
+    formatVersion AS `aws.waf.formatVersion`,
+    webaclId AS `aws.waf.webaclId`,
+    terminatingRuleId AS `aws.waf.terminatingRuleId`,
+    terminatingRuleType AS `aws.waf.terminatingRuleType`,
+    action AS `aws.waf.action`,
+    httpSourceName AS `aws.waf.httpSourceName`,
+    httpSourceId AS `aws.waf.httpSourceId`,
+    ruleGroupList AS `aws.waf.ruleGroupList`,
+    rateBasedRuleList AS `aws.waf.rateBasedRuleList`,
+    nonTerminatingMatchingRules AS `aws.waf.nonTerminatingMatchingRules`,
+    requestHeadersInserted AS `aws.waf.requestHeadersInserted`,
+    responseCodeSent AS `aws.waf.responseCodeSent`,
+    httpRequest AS `aws.waf.httpRequest`,
+    labels AS `aws.waf.labels`,
+    captchaResponse AS `aws.waf.captchaResponse`,
+    challengeResponse AS `aws.waf.challengeResponse`,
+    ja3Fingerprint AS `aws.waf.ja3Fingerprint`
+FROM
+    {table_name}
+;
+
+REFRESH MATERIALIZED VIEW mys3.default.{table_name}_mview
```

Comment:
@seankao-az for this line, should we modify this into:
```sql
...
WITH (
  auto_refresh = 'true',
  checkpoint_location = '{s3_bucket_location}',
  watermark_delay = '1 Minute',
  extra_options = '{ "{table_name}": { "maxFilesPerTrigger": "10" }}'
);
```

Reply:
sure. go ahead